newsroom/auth_server/oauth2.py

```diff
@@ -44,10 +45,8 @@ async def issue_token():
         raise
     else:
         if client_id:
-            client = get_cached_resource_by_id("oauth_clients", client_id)
-            superdesk.get_resource_service("oauth_clients").system_update(
-                ObjectId(client_id), {"last_active": current_time}, client
-            )
+            client = await ClientService().find_by_id(client_id)
+            await ClientService().system_update(ObjectId(client_id), {"last_active": current_time}, client)
```

Comment:
I think the `client` is wrong, if I'm not mistaken `system_update` takes only two params.
```suggestion
            await ClientService().system_update(ObjectId(client_id), {"last_active": current_time})
```